src/subsai/models/whisper_api_model.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Whisper API Model
+
+See [openai/whisper](https://platform.openai.com/docs/guides/speech-to-text)
+"""
+
+import os
+import ffmpeg
+import tempfile
+from subsai.models.abstract_model import AbstractModel
+from subsai.utils import _load_config
+from openai import OpenAI
+from pysubs2 import SSAFile
+from pydub import AudioSegment
```

Comment:
I think we will need to add `pydub` to the requirements in that case as well ?

Reply:
We can add it just to be safe (if that dependency ever decided to remove `pydub` down the road) :)
But it's OK for now.